bitcoinplusplus/layer-2/exploring-p2p-hashrate-markets-v2.md

```diff
@@ -218,44 +236,52 @@ And cloud mining, you're getting price gouged almost always.
 Oh, wow, okay.
 That was fast.
 Well, I'm gonna run through this really quick.
 Thank you.
+
+
+## P2P Hashrates
+
 So our model is where you go, we have auctions set up in order to have price discovery.
 We have multi-sig where the payment to the seller of the hash rate, or the miner, only occurs after the hash rate's been delivered, and we deliver the hash rate through a proxy.
 So we're able to see exactly how much hash rate ends up getting delivered.
 So Here we have a little model of how this works.
 So we create this little agreement here in the middle.
 The agreement, the purchase part here in orange at the bottom is held on chain in multisig.
 The delivery for The terms of delivery are held through the pool account.
-So the hash rate gets pointed here from the miner to a pool where eventually a coinbase hits the pool and the payout goes to a mining pool account, right?
+So the hash rate gets pointed here from the miner to a pool where eventually a coinbase hits the pool and the payout goes to a mining pool account.
 And that's where this part of the agreement comes into play.
-And finally, you can actually obviously remove your funds from your account model to the next model, right?
+And finally, you can actually obviously remove your funds from your account model to the next model.
 So you have your hot dog shape.
 And the way in which we actually are able to ensure that that delivery is happening is that we have a proxy that we can read that the hash rate is getting delivered to that pool account.
 So we have a demo running right now, which you can sign up using this QR code, which I'll give another link to in a second.
-This is the basic flow of what's happening with our model, but basically we have an event-driven system where as soon as you make a payment to us, we'll switch a hash rate that is currently pointed elsewhere on our proxy straight to your guys' pool account, the pool account that you have registered with us and our database.
+This is the basic flow of what's happening with our model, but basically, we have an event-driven system where as soon as you make a payment to us, we'll switch a hash rate that is currently pointed elsewhere on our proxy straight to your guys' pool account, the pool account that you have registered with us and our database.
+
+
+## Lightning 
+
 Last notes on lightning, we want to be able to bring mining pool payouts over lightning.
 The issue with that is that, I mean this is, I don't know if you guys saw the Jade, basically it produces hashes.
-And the idea here is like, okay, well, we have 10 sats per terahash per second per hour.
+And the idea here is like, we have 10 sats per terahash per second per hour.
 How would you be able to stream that over lightning?
 Would that be possible?
 Could we use every spare or wasted jewel and convert that into hashes or gigahashes or whatever it may be?
```

Comment:
Not "jewel" but "joule" the unit of energy
```suggestion
Could we use every spare or wasted joule and convert that into hashes or gigahashes or whatever it may be?
```